dojo/tools/nexpose/parser.py

```diff
@@ -184,6 +185,14 @@ def get_items(self, tree, vulns, test):
                 host['services'] = list()
                 host['vulns'] = self.parse_tests_type(node, vulns)
 
+                host['vulns'].append({
+                    'name': 'Host Up',
+                    'desc': 'Host is up because it replied on ICMP request or some TCP/UDP port is up',
+                    'severity': 'Info',
+                    'resolution': 'N/A',
+                    'vector': 'No impact provided',
```

Comment:
Not fan of 'N/A' and 'No xxx provided'. We have null-able attributes so, we can find a way to not set them if there is no data.

Reply:
Fixed https://github.com/DefectDojo/django-DefectDojo/pull/4564/commits/506033e219c51e6e3fa15ae64ce2124fc59f1a34